install/helm/agones/values.yaml

```diff
@@ -99,6 +99,12 @@ agones:
     pdb:
         minAvailable: 1
     replicas: 2
+    readiness:
+      initialDelaySeconds: 3
+      periodSeconds: 3
+      failureThreshold: 3
+    sleepPeriod: 15
+    sleepFailurThreshold: 2
```

Comment:
I think you may have gotten the two sleep variables off my previous suggestion about the variables _here_. 

I'm not sure the shutdown duration should make it to values.yaml. Instead, I think that you should remove it here, but in `extensions-deployment.yaml`, set:

```
- SHUTDOWN_DURATION
  value: {{ .Value.agones.extensions.periodSeconds * .Value.agones.extensions.failureThreshold * 2 }}s
```

That's where I was suggesting a multiplication.

The point is, we might want to refine the failure thresholds / readiness checking for some issue, but in general, we just need some safety factor beyond the readiness-failure duration (which is where the 2* is from) to drain. The user doesn't need to configure that part, I don't think.

Reply:
Ooh I see, I did not realize you could do the multiplication here.

Reply:
you can't in the values.yaml, but in the deployment yaml, sure.

Reply:
done!